Source/FieldSolver/FiniteDifferenceSolver/MacroscopicEvolveHM_2nd.cpp

```diff
@@ -943,8 +977,9 @@ void FiniteDifferenceSolver::MacroscopicEvolveHMCartesian_2nd(
 
                         [=] AMREX_GPU_DEVICE(int i, int j, int k) {
 
-                            Real mag_Ms_arry = MacroscopicProperties::macro_avg_to_face(i,j,k,Myface_stag,mag_Ms_arr);
-
+                            amrex::Real x, y, z;
+                            WarpXUtilAlgo::getCellCoordinatesIntVect(i, j, k, Myface_stag, problo, dx, x, y, z);
```

Comment:
We could use the GpuArray for My_stag here as well?

Reply:
Discussed this during the meeting. Since Myface_stag is same as My_stag, we can use the same getCellCoordinate function for GPY array type mf_type variable.